docs/reference/sql/language/data-types.asciidoc

```diff
@@ -7,7 +7,10 @@ beta[]
 
 Most of {es} <<mapping-types, data types>> are available in {es-sql}, as indicated below.
 As one can see, all of {es} <<mapping-types, data types>> are mapped to the data type with the same
```

Comment:
This needs to be better emphasized - in asciidoc this can be achieve through admonitions (https://asciidoctor.org/docs/asciidoc-writers-guide/#admonitions)

Reply:
AS `NOTE` or `IMPORTANT` ?

Reply:
I would pick `NOTE` since it's just FYI